packages/bruno-electron/src/ipc/network/prepare-request.js

```diff
@@ -204,8 +205,7 @@ const prepareRequest = (request, collectionRoot, collectionPath) => {
       axiosRequest.headers['content-type'] = 'application/json';
     }
     try {
-      // axiosRequest.data = JSON.parse(request.body.json);
-      axiosRequest.data = JSON.parse(decomment(request.body.json));
+      axiosRequest.data = JSONbig.parse(decomment(request.body.json));
```

Comment:
This also needs to be changed for the CLI

Reply:
Added for the CLI too. Thanks for the review @Its-treason